torch/_dynamo/variables/tensor.py

```diff
@@ -352,7 +352,7 @@ def call_method(
             elif dim_var is not None:
                 dim = dim_var.as_python_constant()
 
-            def make_const_size_variable(x, **options):
+            def make_const_size_variable(x, **options) -> SizeVariable:
```

Comment:
mypy gives error here. Since we use a fixed mypy version, I haven't tried a newer version but directly modified code to make mypy==0.960 happy.
```
/home/lkct/pytorch/torch/_dynamo/variables/tensor.py:318: error: INTERNAL ERROR -- Please try using mypy master on GitHub:
https://mypy.readthedocs.io/en/stable/common_issues.html#using-a-development-mypy-build
If this issue continues with mypy master, please report a bug at https://github.com/python/mypy/issues
version: 0.960
```